Classes/Settings/SettingsViewController.swift

```diff
@@ -134,6 +137,14 @@ NewIssueTableViewControllerDelegate {
             else { fatalError("Should always create GitHub Status URL") }
         presentSafari(url: url)
     }
+  
+    func onReivewOnAppStore() {
+      guard let url = URL(string: "itms-apps://itunes.apple.com/us/app/githawk-for-github/id1252320249")
+        else { fatalError("Should always be valid app store URL") }
+      if UIApplication.shared.canOpenURL(url) {
+        UIApplication.shared.open(url, options: [:], completionHandler: nil)
```

Comment:
Nit: you can omit the `options` and `completionHandler` parameter as they default to `[:]` and `nil` respectively.

Reply:
Makes sense.